docs/en/Alluxio-Storage.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: global
+title: Alluxio Storage
+nickname: Alluxio Storage
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+Alluxio manages the local storage, including memory, of Alluxio workers to act as a distributed
+buffer cache. This fast data layer between user applications and the various under storages results
+in vastly improved I/O performance.
+
+The amount and type of storage for each Alluxio node to manage is determined by user configuration.
+Alluxio also supports tiered storage, which makes the system storage media aware, enabling data
+storage optimizations similar to L1/L2 cpu caches.
+
+## Configuring Alluxio Storage
+
+The easiest way to configure Alluxio storage is to use the default single-tier mode.
+
+Out-of-the-box Alluxio will provision a ramdisk on every worker and take a percentage of the
+system's total memory. This ramdisk will be used as the only storage medium allocated to each
+Alluxio worker.
+
+Alluxio storage is configured through Alluxio's configuration in `alluxio-site.properties`. See the
+[configuration docs](Configuration-Settings.html) for detailed information.
+
+A common modification to the default is to explicitly set the ramdisk size. For example, to set the
+ramdisk size to be 16GB on each worker:
+
+```
+alluxio.worker.memory.size=16GB
+```
+
+Another common change is to specify multiple storage media, such as ramdisk and SSDs. We will need
+to update `alluxio.worker.tieredstore.level0.dirs.path` to take specify each storage medium we want
```

Comment:
discussion: can we actually introduce two more new conf properties, e.g., `alluxio.worker.storage.path` and `alluxio.worker.storage.quota`, which will be the default value for `alluxio.worker.tieredstore.level0.dirs.path` and `alluxio.worker.tieredstore.level0.dirs.quota`. In this way, if users don't really use storage media in ties, users don't need to deal with the complicated conf `alluxio.worker.tieredstore.level{x}.*`, which can be completely hidden into the tiered storage subsection---not really used in common setups.

Reply:
Setting the defaults among three variables may be tricky, I will give it try in a separate PR.

Reply:
SG